apps/user_ldap/lib/Command/UpdateUUID.php

```diff
@@ -49,45 +49,29 @@ class UuidUpdateReport {
 	public const UNWRITABLE = 4;
 	public const UNMAPPED = 5;
 
-	public $id = '';
-	public $dn = '';
-	public $isUser = true;
-	public $state = self::UNCHANGED;
-	public $oldUuid = '';
-	public $newUuid = '';
-
-	public function __construct(string $id, string $dn, bool $isUser, int $state, string $oldUuid = '', string $newUuid = '') {
-		$this->id = $id;
-		$this->dn = $dn;
-		$this->isUser = $isUser;
-		$this->state = $state;
-		$this->oldUuid = $oldUuid;
-		$this->newUuid = $newUuid;
+	public function __construct(
+		public string $id = '',
+		public string $dn = '',
+		public bool $isUser = true,
+		public int $state = self::UNCHANGED,
```

Comment:
```suggestion
		public string $id,
		public string $dn,
		public bool $isUser,
		public int $state,
```
These should stay mandatory parameters